src/model.ts

```diff
@@ -17,7 +18,7 @@ interface Game {
 interface Player {
   name: string;
   playerId: string;
-  // add time here
+  timer: any;
```

Comment:
# Important: This methodology will not work, as a player can be part of more than one game

We should consider adding timer as an object inside the game object something like the following

```typescript
const game = {
  'game-id': {
    ...gameProperties,
    timers: {
      'player-id-1': 5000,
      'player-id-2': 6000, // player time in ms
    }
  }
}
```

Also general rule of thumb - never use `any`